pkgs/dart_mcp/lib/src/api/tools.dart

```diff
@@ -300,6 +324,13 @@ extension type ValidationError.fromMap(Map<String, Object?> _value) {
 /// if you need something more complex you can create your own
 /// `Map<String, Object?>` and cast it to [Schema] (or [ObjectSchema]) directly.
 extension type Schema.fromMap(Map<String, Object?> _value) {
+  factory Schema({JsonType? type, String? title, String? description}) =>
```

Comment:
See line 355 for what I meant - I don't think this unnamed constructor makes sense

Reply:
Did you mean this one?:

```
  /// Alias for [EnumSchema.new].
  static const enumeration = EnumSchema.new;
```

That was already there, which is why I got confused...

Reply:
Yeah that is what I meant, I guess I just missed that